libraries/standard/mqtt/src/mqtt.c

```diff
@@ -1403,6 +1411,9 @@ MQTTStatus_t MQTT_Init( MQTTContext_t * pContext,
         pContext->appCallback = userCallback;
         pContext->networkBuffer = *pNetworkBuffer;
 
+        /* Set ping response timeout for process loop. */
+        pContext->pingRespTimeoutMs = MQTT_DEFAULT_PINGRESP_TIMEOUT_MS;
```

Comment:
If we are adding a config macro for the ping response timeout, I think that the timeout should apply across all MQTT connections. 
If the ping response timeout is specific to the connection, then it should be an input parameter to `MQTT_Init`

Reply:
Removed the field from the context and made it just a library wide macro